runtime/service.go

```diff
@@ -499,6 +500,13 @@ func (s *service) createVM(requestCtx context.Context, request *proto.CreateVMRe
 // created yet and the timeout is hit waiting for it to exist, an error will be returned but the shim will
 // continue to shutdown.
 func (s *service) StopVM(requestCtx context.Context, request *proto.StopVMRequest) (_ *empty.Empty, err error) {
+	var timeout time.Duration
```

Comment:
One minor nit commit, but otherwise looks good.

```go
timeout := defaultStopVMTimeout
if request.VMShutdownTimeoutSeconds > 0 {
    timeout = time.Duration(request.VMShutdownTimeoutSeconds) * time.Second
}
```